<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSelect_configuration"/>
    <addaction name="separator"/>
    <addaction name="actionDll"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAuthors"/>
   </widget>
   <widget class="QMenu" name="menuTracer">
    <property name="title">
     <string>Tracer</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionTracePoints"/>
    <addaction name="actionTaintSources"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="actionNOPSections"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTracer"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAdvanced"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSelect_configuration"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionTaintSources"/>
   <addaction name="actionTracePoints"/>
   <addaction name="actionNOPSections"/>
   <addaction name="actionDll"/>
  </widget>
  <action name="actionDll">
   <property name="icon">
    <iconset theme="audio-volume-medium" resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-dll-26.png</normaloff>:/res/res/icons8-dll-26.png</iconset>
   </property>
   <property name="text">
    <string>DLL selector</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-exit-26.png</normaloff>:/res/res/icons8-exit-26.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-opened-folder-26.png</normaloff>:/res/res/icons8-opened-folder-26.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSelect_configuration">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-log-26.png</normaloff>:/res/res/icons8-log-26.png</iconset>
   </property>
   <property name="text">
    <string>PIN configuration</string>
   </property>
   <property name="toolTip">
    <string>PIN configuration</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-start-26.png</normaloff>:/res/res/icons8-start-26.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-stop-26.png</normaloff>:/res/res/icons8-stop-26.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionTracePoints">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-flag-filled-26.png</normaloff>:/res/res/icons8-flag-filled-26.png</iconset>
   </property>
   <property name="text">
    <string>Tracepoints</string>
   </property>
  </action>
  <action name="actionTaintSources">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-select-26.png</normaloff>:/res/res/icons8-select-26.png</iconset>
   </property>
   <property name="text">
    <string>Taint sources</string>
   </property>
  </action>
  <action name="actionNOPSections">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/icons8-knight-26.png</normaloff>:/res/res/icons8-knight-26.png</iconset>
   </property>
   <property name="text">
    <string>NOP sections</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="icon">
    <iconset resource="MainToolBarResource.qrc">
     <normaloff>:/res/res/appicon.png</normaloff>:/res/res/appicon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainToolBarResource.qrc"/>
 </resources>
 <connections/>
</ui>
